### DISCUSSION TITLE
Can't get bold, italic etc. to work

### DISCUSSION THREAD

**Discussion by winslowb** [nvim-neorg/neorg#761] [Q&A]
Howdy. I've not been successful using the neorg formatting. The exception being _bullet_. 
![image](https://user-images.githubusercontent.com/71040996/221338901-126c8dac-c277-441b-9842-f106a3a77b88.png)

I followed the packer config instructions
   use {                                                                                    
     'nvim-telescope/telescope.nvim',                                                        
     tag = '0.1.0',                                                                          
     requries = { {'nvim-lua/plenary.nvim'} }                                                
    }                                                                                        
    use {                                                                                    
     "nvim-neorg/neorg",                                                                    
       config = function()                                                                    
           require('neorg').setup {                                                           
               load = {                                                                       
                   ["core.defaults"] = {}, -- Loads default behaviour                         
                   ["core.norg.concealer"] = {}, -- Adds pretty icons to your documents       
                   ["core.norg.dirman"] = { -- Manages Neorg workspaces                       
                       config = {                                                             
                           workspaces = {                                                     
                              notes = "~/notes",                                             
                           },                                                                 
                       },                                                                     
                   },                                                                         
               },                                                                             
           }

**Comment by jamiereid** [comment]
Hi @winslowb - have a read through the thread for issue #690. The key thing for me was ensuring that nvim-treesitter was configured correctly, and that `:Neorg sync-parsers` was run.

**Comment by miraculusik** [comment]
can you try this code in nvim config
```lua
vim.opt.conceallevel = 2 -- btw 0-3
vim.opt.concealcursor = "nc"
-- more info
-- :h conceallevel
-- :h concealcursor
```

**Reply by miraculusik** [reply]
if you are using macos ventura or apple silicon mac look at this guide. Maybe can help you
https://github.com/nvim-neorg/neorg/discussions/754#discussion-4872797

**Reply by Ramilito** [reply]
I'm using both macos ventura and apple silicon but not getting bold/italic and such to work, followed the guide as well and still not working.
I feel that everything else is working though, icons, bullet-list linking files and so on.

**Reply by miraculusik** [reply]
maybe it's about the color scheme or terminal app. Im using [Alacritty](https://github.com/alacritty/alacritty) and [Kanagawa](https://github.com/rebelot/kanagawa.nvim) color scheme with no issues.

**Reply by satk0** [reply]
@miraculusik For me it was indeed Neovim color scheme issue. It is important to run Neorg on color schemes with TreeSitter support like mentioned Kanagawa or [gruvbox.nvim](https://github.com/ellisonleao/gruvbox.nvim)
